hack/bazel-build.sh

```diff
@@ -28,6 +28,7 @@ mkdir -p ${CMD_OUT_DIR}/dump
 
 # Build all binaries for amd64
 bazel build \
+    --host_javabase="@local_jdk//:jdk" \
```

Comment:
This seems to improve things drastically. The choice is also good, because it uses the embedded jdk from bazel (again).

Can you however instead of putting it here, add to `.bazelrc`? So that we pick this setting transparently up?

Reply:
oh, that's a much nicer place!